src/main/java/io/snyk/maven/plugins/SnykTest.java

```diff
@@ -292,6 +297,17 @@ private void processVulns(JSONObject responseJson) throws MojoFailureException {
         }
     }
 
+    private boolean isIssueFixable(JSONObject vuln) {
+        boolean upgradable =
+            (vuln.get("isUpgradable") != null && (boolean) vuln.get("isUpgradable"));
+        boolean fixable = false;
+        if (vuln.get("fixedIn") != null) {
+            JSONArray fixedIn = (JSONArray) vuln.get("fixedIn");
+            fixable = (!fixedIn.isEmpty());
+        }
+        return upgradable || fixable;
+    }
+
```

Comment:
An issue is fixable if it can be eliminated with an upgrade ("isUpgradable") or patch ("isPatchable").